### ISSUE TITLE
Warning in ALFRESCO endpoint?

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/data-api#458]
```
Sep 12 15:11:43 ip-172-31-4-138 web: /var/app/current/routes/alfresco.py:149: RuntimeWarning: Mean of empty slice
 Sep 12 15:11:43 ip-172-31-4-138 web: results = np.nanmean(data_arr, axis=(1, 2)).astype(float)
```

**Comment by charparr** [comment]
@Joshdpaul @BobTorgerson I assume this can be closed now as well as it is effectively a dupe of the now closed #272?

**Comment by BobTorgerson** [comment]
Yes, I agree that Josh's explanation in #272 applies here and this can be safely closed.
